static/docs/user-guide/data-sharing/mounted-cache.md

```diff
@@ -0,0 +1,144 @@
+# Sharing Data Through a Mounted Cache
+
+We have seen already how to share data through a
+[mounted DVC storage](/doc/user-guide/data-sharing/mounted-storage). In that
+case we have a copy of the data on the DVC storage and at least one copy on each
+user project, since deduplication does not work across filesystems.
+
+However the data management can be further optimized if we use a shared cache.
+The idea is that instead of mounting the DVC storage from the server, we can
+directly mount the cache directory (`.dvc/cache/`). If all the users do this,
+then effectively they will be using the same cache directory (which is mounted
+from the NAS server). So, if one of them adds something to the cache, it will
+appear automatically to the cache of all the others. As a result, no `dvc push`
+and `dvc pull` are needed to share the data, just a `dvc checkout` will be
+sufficient.
+
+> ** ❗ Caution:** Deleting data from the cache will also make it disappear from
+> the cache of the other users. So, be careful with the command `dvc gc` (which
+> cleans obsolete data from the cache) and consult the other users of the
+> project before using this command.
+
+The optimization in data management comes from using the _symlink_ cache type.
+You can find more details about it in the page of
+[Large Dataset Optimization](https://dvc.org/doc/user-guide/large-dataset-optimization).
+
+## Mounted Cache Example
+
+In this example we will see how to share data with the help of a cache directory
+that is mounted through SSHFS. We are using a SSHFS example because it is easy
+to network-mount a directory with SSHFS. However once you understand how it
+works, it should be easy to implement it for other types of network-mounted
+storages (like NFS, Samba, etc.).
+
+> For more detailed instructions check out this
+> [interactive example](https://katacoda.com/dvc/courses/examples/mounted-cache).
+
+<p align="center">
```

Comment:
that's not what we usually do I believe (not sure why) to center the images ... is there a special Markdown syntax to center an image? In general it's better to keep as less HTML as possible.

Reply:
> that's not what we usually do I believe (not sure why) to center the images ... 

You are using very large images, which are scaled down by the browser and so they are displayed on the center of the page. I am not sure whether this is a good or bad practice...

> is there a special Markdown syntax to center an image?

As far as I know there isn't.

> In general it's better to keep as less HTML as possible.

I certainly agree with this.